Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp

```diff
@@ -816,9 +816,9 @@ void MediaPlayerPrivateGStreamerMSE::dispatchDecryptionKey(GstBuffer* buffer)
 #endif
 
 #if USE(PLAYREADY)
-void MediaPlayerPrivateGStreamerMSE::emitSession()
+void MediaPlayerPrivateGStreamerMSE::emitPlayReadySession()
 {
-    fprintf(stderr, "MediaPlayerPrivateGStreamerMSE::emitSession()\n");
+    fprintf(stderr, "MediaPlayerPrivateGStreamerMSE::emitPlayReadySession()\n");
```

Comment:
It was here before you touched this, but it's a good oportunity to make this a GST_TRACE. You don't need the function at the message because it will be added by GST_TRACE so it would be something like "emitting session".